kuma/static/styles/includes/_mixins.scss

```diff
@@ -527,6 +516,32 @@ Placeholders
     @include vendorize(hyphens, none);
 }
 
+%cta-link {
+    position: relative;
+    box-sizing: border-box;
+    display: inline-block;
+    border-bottom: 2px solid;
+    padding-bottom: 10px;
+    @include bidi-style(padding-right, 30px, padding-left, 0);
+    @include bidi-value(text-align, left, right);
+
+    &:after {
+        position: absolute;
+        bottom: 10px;
+        @include bidi-style(right, 0, left, auto);
+        @include bidi-value(content, '\f061', '\f060');
+        font-family: fontAwesome;
+    }
+
+    &:hover,
+    &:focus {
+        text-decoration: none;
+
+        &:after {
+            // tranform to move arrow
```

Comment:
Oops, no that's me not putting TODO in front of it so I didn't find it when making sure I'd don all my TODOs >_<